CheckSequence.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Analyzes the sequence that was entered by the user.
+ */
+class CheckSequence {
+/**
+ * Determines whether the sequence is non-decreasing,
+ * and displays the result on the screen.
+ * @param gaps the sequence that was entered by the user
+ *             and that must be checked.
+ */
+  public void checkNondecreasing(String[] gaps) {
+    try {
+      for (int i = 1; i < gaps.length; i++) {
+        if (Integer.parseInt(gaps[i]) < Integer.parseInt(gaps[i-1])) {
+          System.out.println("Your numerical sequence is not non-decreasing.");

```

Comment:
single responsibility violation - method must check (1 responsibility), not "check, verify & output" (3 responsibility)